packages/config/config/devices/0x001d/dz6hd.json

```diff
@@ -16,131 +16,36 @@
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Fade On Time",
-			"description": "Values 1-127 = seconds; 128-253 = minutes (minus 127)",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 253,
-			"defaultValue": 2,
-			"unsigned": true,
-			"options": [
-				{
-					"label": "Instant on",
-					"value": 0
-				}
-			]
+			"$import": "templates/config_template.json#fade_on_time"
 		},
 		{
 			"#": "2",
-			"label": "Fade Off Time",
-			"description": "Values 1-127 = seconds; 128-253 = minutes (minus 127)",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 253,
-			"defaultValue": 2,
-			"unsigned": true,
-			"options": [
-				{
-					"label": "Instant off",
-					"value": 0
-				}
-			]
+			"$import": "templates/config_template.json#fade_off_time"
 		},
 		{
 			"#": "3",
-			"label": "Minimum Light Level",
-			"valueSize": 1,
-			"unit": "%",
-			"minValue": 0,
-			"maxValue": 100,
+			"$import": "~/templates/master_template.json#minimum_dim_level_1-99",
 			"defaultValue": 10
 		},
 		{
 			"#": "4",
-			"label": "Maximum Light Level",
-			"valueSize": 1,
-			"unit": "%",
-			"minValue": 0,
-			"maxValue": 100,
-			"defaultValue": 100
+			"$import": "~/templates/master_template.json#maximum_dim_level_0-100"
 		},
```

Comment:
These stand out a bit, since Z-Wave usually has levels of 0/1...99. Do the devices actually accept 100 as a value? Is it in % then?

Reply:
I have checked multiple Leviton device documents and they do indeed accept 0-100 as values. (I specifically looked at this device Leviton DZ6HD)